master/pkg/schemas/expconf/storage_config.go

```diff
@@ -136,3 +138,12 @@ type S3ConfigV0 struct {
 type GCSConfigV0 struct {
 	RawBucket *string `json:"bucket"`
 }
+
+//go:generate ../gen.sh
+// AzureConfigV0 configures storing checkpoints on Azure.
+type AzureConfigV0 struct {
```

Comment:
You should implement the schemas.Mergable interface so that this business of a custom connection string or account url is preserved through a merge operation.